modules/api/pkg/provider/cloud/kubevirt/networks.go

```diff
@@ -56,7 +57,8 @@ func GetProviderNetworkSubnets(ctx context.Context, client ctrlruntimeclient.Cli
 
 	subnets := make([]string, 0, len(subs))
 	for _, subnet := range subs {
-		subnets = append(subnets, subnet.Name)
+		subnetName := fmt.Sprintf("%v (%v)", subnet.Name, subnet.CIDRBlock)
```

Comment:
I think we should handle this change on frontend because by changing value of `name`, frontend will send this exact value to backend which might cause problems.

Reply:
Currently, only `name` field is sent in response by API so we'll need to add this `cidrblock` field to response.

Reply:
Ah yeah good catch! Let me take care of it.